docs/python/tutorial-flask.md

```diff
@@ -390,7 +358,7 @@ The following sections demonstrate both types of static files.
 
     ```html
     {%if name %}
-        <span class="message">Hello there, \{{ name }}!</span> It's \{{ date.strftime("%A, %d %B, %Y at %X") }}.
```

Comment:
We need to leave the backslash escape characters here.
It's a Gotcha that hurks page rendering https://github.com/microsoft/vscode-docs/blob/main/CONTRIBUTING.md#double-opening-curly-braces-break-generated-handlebar-files

Reply:
reverted